src/components/DropdownFilterModal/DropdownFilterInput.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, FunctionComponent } from "react";
+import { StyleSheet, View, TextInput } from "react-native";
+import { size, borderRadius, color, fontSize } from "../../common/styles";
+import { AppText } from "../Layout/AppText";
+import { DropdownFilterModal, DropdownItem } from "./DropdownFilterModal";
+
+const styles = StyleSheet.create({
+  label: {
+    fontFamily: "brand-bold"
+  },
+  inputView: {
+    marginTop: size(1),
+    minHeight: size(6),
+    borderRadius: borderRadius(2),
+    borderWidth: 1,
+    backgroundColor: color("grey", 0),
+    borderColor: color("blue", 50)
+  },
+  inputText: {
+    minHeight: size(6),
+    paddingHorizontal: size(1),
+    fontSize: fontSize(0),
+    color: color("blue", 50)
+  }
+});
+
+export interface DropdownFilterInput {
+  label: string;
+  placeholder: string;
+  value?: string;
+  dropdownItems: DropdownItem[];
+  onItemSelection: (item: DropdownItem) => void;
+}
+
+export const DropdownFilterInput: FunctionComponent<DropdownFilterInput> = ({
+  label,
+  placeholder,
+  value,
+  dropdownItems,
+  onItemSelection
+}) => {
+  const [modalVisible, setModalVisible] = useState(false);
+
+  const closeModal = (): void => {
+    setModalVisible(false);
+  };
```

Comment:
Minor thing, noticed we extracted this but not `openModal`, do we need `openModal`?

Reply:
Yup I have added `openModal` so that the api looks more consistent